Begin
`a` has been called!
1
catch(@e) where {: see(I, $x) :}
ERROR {: #92cfb9b3-3b45-416b-b35f-08e5b484f575 &gt;: { see(i,#a) } :}
on Fired
ensure
#a
End of `Enter`
